modules/cloudaccess/wf_cluster_manager.tf

```diff
@@ -13,6 +13,20 @@ resource "azurerm_role_definition" "clustermanager" {
   }
 }
 
+resource "time_sleep" "after_azurerm_role_definition_clustermanager" {
```

Comment:
You could technically just have one of these resources `after_azurerm_role_definition`, with the `depends_on` list including all role definition resources. Even if those role definitions are conditional and may not be created, the `depends_on` will just skip that resource if it's not created so it doesn't cause issues.

Reply:
Yeah, I thought about this - to me it felt cleaner to make things depend only on the things they need, i.e. the assignment depends ONLY on its own role, not all of them. other than the repetition, is there any issue with what's been done here?

Reply:
No issues, I'm fine with that 😊